contracts/TokenMigration.aes

```diff
@@ -1,9 +1,14 @@
 payable contract TokenMigration =
 
-    datatype event = Migrated(bytes(20), address, int)
+    datatype event = Migrated(string, address, int)
```

Comment:
You could also make arguments `indexed`.

Reply:
The `indexed` keyword doesn't do anything (all word types are indexed automatically). See
https://github.com/aeternity/aesophia/blob/master/CHANGELOG.md#310---2019-06-03